src/aiohomeconnect/model/event.py

```diff
@@ -347,6 +347,10 @@ def _missing_(cls, value: object) -> EventKey:
         OptionKey.DISHCARE_DISHWASHER_VARIO_SPEED_PLUS
     )
     DISHCARE_DISHWASHER_OPTION_ZEOLITE_DRY = OptionKey.DISHCARE_DISHWASHER_ZEOLITE_DRY
+    HEATING_VENTILATION_AIR_CONDITIONING_AIR_CONDITIONER_FAN_SPEED_MODE = (
+        OptionKey.HEATING_VENTILATION_AIR_CONDITIONING_AIR_CONDITIONER_FAN_SPEED_MODE
+    )
+    HEATING_VENTILATION_AIR_CONDITIONING_AIR_CONDITIONER_FAN_SPEED_PERCENTAGE = OptionKey.HEATING_VENTILATION_AIR_CONDITIONING_AIR_CONDITIONER_FAN_SPEED_PERCENTAGE
```

Comment:
Try disabling the ruff check for line length on this line (after using a parenthesis and newline after the equal sign).